spec/acceptance/nodesets/sles-12-x64.yml

```diff
@@ -0,0 +1,10 @@
+HOSTS:
+  sles-12-x64:
+    roles:
+      - default
+    platform: sles-12-x64
+    box: suse/sles12sp1
+    box_url: https://vagrantcloud.com/suse/boxes/sles12sp1
```

Comment:
I think this is redundant, no?
Does this work for you? For me, I get an error on mounting `/vagrant`
```
RuntimeError:
  Failed to exec 'vagrant up'. Error was Vagrant was unable to mount VirtualBox shared folders. This is usually
  because the filesystem "vboxsf" is not available. This filesystem is
  made available via the VirtualBox Guest Additions and kernel module.
  Please verify that these guest additions are properly installed in the
  guest. This is not a bug in Vagrant and is usually caused by a faulty
  Vagrant box. For context, the command attempted was:
  
  mount -t vboxsf -o uid=1000,gid=1000 vagrant /vagrant
  
  The error output from the command was:
  
  mount: unknown filesystem type 'vboxsf'
```

Reply:
ps - there's also a `sles12sp2` one now. I'm proposing an update in modulesync_config that should provide roughly the same stuff.

Reply:
From what I can tell, there's only OpenSUSE docker nodesets now (with 42.x being equivalent to 12spX), but then opensuse doesn't seem to be a valid platform for Beaker.

Reply:
@wyardley Thanks for taking a look. Using the existing `sles12sp2` and the OpenSUSE docker nodesets makes sense to me. I didn't see one for `sles12sp2` in modulesync. Has this been updated, or is this a proposed change we are looking to make to modulesync? I suspect we will have to use OpenSUSE for this due to the licensing and repo restrictions.

Reply:
@jarretlavallee 
I checked, and it doesn't look like opensuse is currently supported by Beaker AFAICT; the difference in version numbers may also be a hassle.
Can you test if the Vagrant nodesets here work for you? Very similar to what you had, I just updated a version and removed the URL.
https://github.com/voxpupuli/modulesync_config/pull/405/files

Reply:
@wyardley I tested out the nodesets in https://github.com/voxpupuli/modulesync_config/pull/405/files, but they do not include the virtualbox tools either. Using ` synced_folder: disabled` may be the way we can configure these without mounting the shared folder, however the image does not have any zypper repos configured. I figure that we can use the `http://download.opensuse.org/distribution/leap/42.2/repo/oss/` repo for SLES 12, however I am not sure how to configure this repo prior to other commands being run except for modifying beaker.

I did find https://github.com/puppetlabs/beaker/pull/1277 on OpenSuse, which removes the installed puppet packages from an existing Opensuse image.